hypertrace-core-graphql-attribute-store/src/main/java/org/hypertrace/core/graphql/attributes/CachingAttributeStore.java

```diff
@@ -36,7 +36,7 @@ class CachingAttributeStore implements AttributeStore {
                 channelRegistry.forAddress(
                     serviceConfig.getAttributeServiceHost(),
                     serviceConfig.getAttributeServicePort()))
-            .withCacheExpiration(Duration.ofMinutes(5))
+            .withCacheExpiration(Duration.ofMinutes(1))
```

Comment:
Change as per the discussion @ https://github.com/hypertrace/attribute-service/pull/152#discussion_r1030438251
```suggestion
            .withCacheExpiration(Duration.ofMinutes(3))
```